### DISCUSSION TITLE
Help! G7 connection error

### DISCUSSION THREAD

**Discussion by cobwebT1D** [NightscoutFoundation/xDrip#3546] [Q&A]
My first G7 took a while to connect, but got there in the end. The 2nd connected really quickly but the sensor failed (filament hadn't been inserted). Now my 3rd one just will not connect. I just have "sensor status: stopped" and brain state: initialising errors. It connected via blue tooth OK. I turned my phone off and on and reconnected the bluetooth. Now I have no readings but it appears to be connected?
![Screenshot_20240624-125454](https://github.com/NightscoutFoundation/xDrip/assets/74460116/fe68f1a8-11ea-42ee-b4e8-7c72fb9185b0)
![Screenshot_20240624-125503](https://github.com/NightscoutFoundation/xDrip/assets/74460116/553098e0-8482-4ce3-afe1-d809d66f4ff4)
Google Pixel 7a, Screenshots attached.

I use Android APS so really need to get this sorted.

**Comment by Navid200** [comment]
These are the instructions:
https://navid200.github.io/xDrip/docs/Dexcom/G7.html

As you can see, you are using an older version of xDrip.  
Can you update and see what happens then?

**Comment by cobwebT1D** [comment]
That's odd, I followed the instructions and updated Xdrip when I switched
to the G7 a couple of weeks ago so I assumed it was up to date. Will try
ahead I guess!

**Reply by Navid200** [reply]
Please don't respond to a GitHub email in order to post here.  It will quote everything whether you want to or not.
Please only quote if needed in order to help readers.

This explains how xDrip update works:
https://navid200.github.io/xDrip/docs/Updates.html
